main/swadge_esp32.c

```diff
@@ -270,6 +270,10 @@ void app_main(void)
     // esp_efuse_set_rom_log_scheme(ESP_EFUSE_ROM_LOG_ALWAYS_OFF);
     esp_efuse_set_rom_log_scheme(ESP_EFUSE_ROM_LOG_ALWAYS_ON);
 
+    /* Initialize USB peripheral */
+    tinyusb_config_t tusb_cfg = {};
+    tinyusb_driver_install(&tusb_cfg);
```

Comment:
I had moved this to `mode_gamepad.c` because, as you pointed out, there may be other modes that want to use the USB-C connector as a UART link cable or something, so I didn't want to initialize the driver for all modes always. Is it possible to move this to your sandbox mode too, or is doing it here necessary?

Reply:
I am confident you will 100% flip to using this for everything.  It gives you a printf terminal, it lets you reboot the swadge by command-line, and start it after flashing.  We should do this.

Reply:
i suppose if i want to co-opt those GPIOs as a UART, we'll cross that bridge when we get there.